packages/devtools/test/utils_test.dart

```diff
@@ -127,24 +127,84 @@ void main() {
     });
 
     test('longestFittingSubstring', () {
-      const str = 'ComponentElement.performRebuild';
+      const asciiStr = 'ComponentElement.performRebuild';
+      const nonAsciiStr = 'ԪԪ';
```

Comment:
nit: a test for a non-ascii string should really contain ascii characters as well as non-ascii characters as that is more representative of the real world case.

Reply:
done.